### PR TITLE
remove redundant footprint filter

### PR DESCRIPTION
**Pull request by mkangia**
## Summary

Started as https://github.com/dimagi/commcare-hq/pull/29582 but then was simply about removing redundant code https://github.com/dimagi/commcare-hq/pull/29582#pullrequestreview-641188896.

## Safety Assurance

- [x] Risk label is set correctly
- [x] All migrations are backwards compatible and won't block deploy
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change
- [x] If QA is part of the safety story, the "Awaiting QA" label is used
- [x] I have confidence that this PR will not introduce a regression for the reasons below

### Safety story

Removing an unused session parameter for formplayer, generated by HQ.

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### CODE REVIEW

**Comment by mkangia** [comment]
Risk label is set as medium based on files but I think this should be no risk, thoughts?

**Comment by mkangia** [comment]
@snopoke 

https://github.com/dimagi/formplayer/search?p=1&q=footprint

Could find footprint only in test files? Just wanted to add this here as additional confidence about it not being used in formplayer.

**Review by snopoke — COMMENTED** [review]
Looking at the formplayer code there are no usages of the `additional_filters` field so you could remove the entire field (I can't find any reference to it's usage in git at all).

**Review by millerdev — APPROVED** [review]
I'm not very familiar with this code, but nice cleanup.

**Review by snopoke — COMMENTED** [review]
Before this get's merged I'd like to update the Formplayer code and tests to ensure that this doesn't break anything.

**Comment by snopoke** [comment]
@mkangia looks good so far: https://github.com/dimagi/formplayer/pull/956

Do you think you could write a happy path test for the `start_session` method that validates the data sent to Formplayer?

**Comment by mkangia** [comment]
Thanks for the formplayer PR @snopoke , added some tests

**Comment by mkangia** [comment]
Added "hold off on merge" for formplayer PR to get merged and deployed first

File: corehq/apps/smsforms/app.py
Comment (mkangia) [inline corehq/apps/smsforms/app.py:24]: `case_for_case_submission` was a left over from https://github.com/dimagi/commcare-hq/pull/19056/files

**Comment by snopoke** [comment]
> Added "hold off on merge" for formplayer PR to get merged and deployed first

Actually I think this should be merged first.

**Comment by mkangia** [comment]
@snopoke 

Oh okay, removed the hold merge flag. Any thoughts on the risk flag? https://github.com/dimagi/commcare-hq/pull/29589#issuecomment-824712506

Should be good to review now.

**Review by snopoke — APPROVED** [review]

**Pull request merged**
